e2e/browser/test-app/app/page.tsx

```diff
@@ -1,13 +1,14 @@
+'use client';
```

Comment:
Pages are server components by default. We want this to be a client component because we rely on `window`.